content/exporters/supported-exporters/Node.js/Jaeger.md

```diff
@@ -40,21 +42,26 @@ npm install @opencensus/exporter-jaeger
 Now let's use the Jaeger exporter:
 
 ```js
-var core = require('@opencensus/core');
-var tracing = require('@opencensus/nodejs');
-var jaeger = require('@opencensus/exporter-jaeger');
+const { logger } = require('@opencensus/core');
```

Comment:
Perhaps let's make this style consistent: in some place we have `const { objectName } = require(<import>)` and in others `const objectName = require(<import>)` for example
```nodejs
const tracing = require('@opencensus/nodejs');
```
while here
```nodejs
const { logger } = require('@opencensus/core');
```

Reply:
```const { objectName } = require(<import>)``` i.e. destructuring import is preferred wherever it is feasible.